--- orginal/keycons.h	2009-04-14 00:16:33.000000000 +0700
+++ keycons.h	2009-06-19 11:46:22.000000000 +0700
@@ -34,7 +34,7 @@
 
 #define CP_US_ANSI 1252
 
-typedef enum {UkTelex, UkVni, UkViqr, UkMsVi, UkUsrIM, UkSimpleTelex} UkInputMethod;
+typedef enum {UkTelex, UkVni, UkViqr, UkMsVi, UkUsrIM, UkSimpleTelex, UkSimpleTelex2} UkInputMethod;
 typedef struct _UnikeyOptions UnikeyOptions;
 
 struct _UnikeyOptions
